logs/debug.log

```diff
@@ -0,0 +1,173 @@
+[27/Apr/2016 17:11:21] DEBUG [django.db.backends:89] (0.003) QUERY = "\n            SELECT name, type FROM sqlite_master\n            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'\n            ORDER BY name" - PARAMS = (); args=None
```

Comment:
Can you remove this file and push another commit ? 
1- `git reset --soft HEAD~1`
2- `git reset HEAD logs/debug.log`
3- `git checkout -- logs/debug.log`
4 `Then git add and git commit`